Introspect/UIKitIntrospectionViewController.swift

```diff
@@ -52,9 +52,16 @@ public struct UIKitIntrospectionViewController<TargetViewControllerType: UIViewC
         return viewController
     }
     
+    /// If you find that the `moveToWindowHandler` is not called in certain situations (which has not been discovered yet),
+    /// you can add code to call the `moveToWindowHandler` in the function body.
```

Comment:
This has come to pass as reported in #194 so I'll be taking over and including the fix here.